keywording/text.xml

```diff
@@ -280,6 +280,12 @@ blank or <c>-</c> sanity-check, so please fix this if it occurs <d/> or ask for
 help to do so.
 </p>
 
+<note>
+If you are changing the package list of a bug after it has been processed,
+<d/> i.e. after NATTkA has CCed arches <d/> you should un-CC all arches so that
+NATTkA CCs the correct new set of arches, in addition to the advice above.
```

Comment:
I'd write "In addition" at the beginning of the paragraph and leave out  the "in addition to the advice above" part.

Reply:
Thanks!